MPC/config/dcps_rapidjson.mpb

```diff
@@ -0,0 +1,10 @@
+project: dcps_ts_defaults {
+
+  dcps_ts_flags += -Grapidjson
+
+  expand(RAPIDJSON_ROOT) {
```

Comment:
This duplicates part of `MPC/config/itl.mpb`, can that one be changed to inherit from this one?

Reply:
I went ahead and created a generic `rapidjson.mpd` which both inherit from.